subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_fota

```diff
@@ -157,9 +153,13 @@ config NRF_CLOUD_FOTA_POLL
 	  to nRF Cloud.
 
 if NRF_CLOUD_FOTA_POLL
-
 module = NRF_CLOUD_FOTA_POLL
 module-str = nRF Cloud FOTA Poll
-source "subsys/logging/Kconfig.template.log_config"
-
 endif # NRF_CLOUD_FOTA_POLL
+
+if NRF_CLOUD_FOTA_POLL || NRF_CLOUD_FOTA
+module = NRF_CLOUD_FOTA
+module-str = nRF Cloud FOTA
+endif # NRF_CLOUD_FOTA_POLL || NRF_CLOUD_FOTA
+
+source "subsys/logging/Kconfig.template.log_config"
```

Comment:
If `log_config` is always sourced, I don't think there is any point in guarding the `module` and `module-str` lines?

I don't think the guard actually does anything, but if it does, then you'll sourcing the log config snippet without updating `module` and `module-str`

Reply:
i don't know.
can you or @plskeggs propose a solution?

Reply:
just remove the if and endif lines -- not necessary